File: Matnas/Common/OrganizationAndFamily.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common
{
    public class OrganizationAndFamily
    {
        public int Id { get; set; }
        public Family Family { get; set; }
        public Organization Organization { get; set; }
        public Category Category { get; set; }
        public string Comments { get; set; }
        public Nullable<System.DateTime> DateAdded { get; set; }
    }
}

File: rmatnas/src/app/Classes/OrganizationAndFamily.ts
import { Organization } from './Organization';
import { Family } from './Family';
import { Category } from './Category';

export class OrganizationAndFamily {
    Id: number;
    Family: Family;
    Organization: Organization;
    Category: Category;
    Comments: string;
    dateAdded: string;

    constructor(idFamily: Family, idOrganization: Organization, idCategory: Category, comments: string, dateAdded: string) {
        this.Comments = comments;
        this.Category = idCategory;
        this.Family = idFamily;
        this.Organization = idOrganization;
        this.dateAdded = dateAdded;

    }
}
